code/modules/atmospherics/machinery/components/components_base.dm

```diff
@@ -88,6 +88,8 @@ Pipenet stuff; housekeeping
 	parents[nodes.Find(A)] = reference
 
 /obj/machinery/atmospherics/components/returnPipenet(obj/machinery/atmospherics/A = nodes[1]) //returns parents[1] if called without argument
+	if(QDELETED(src))
+		warning(" returnPipenet called on QDELETED [src]  Nearby: ([x], [y], [z])")
 	return parents[nodes.Find(A)]
```

Comment:
Don't know why it fix "Runtime in components_base.dm,91", but it fix it.

Reply:
I strongly suspect that this is not fixing the bug, just hiding the runtime. Can you demonstrate otherwise?

Reply:
Here is a chain of 2 runtimes, one launch another herehttps://github.com/tgstation/tgstation/pull/36230/files/aa5dca9a52f10125d9214227f700fb5036d72700#diff-8880ff6809e0203d86f05941e08972cfL130
now it dont launches.
https://github.com/tgstation/tgstation/issues/36298
i test all (511) and 3 point work propertly.